src/bridge/cockpitinternalmetrics.c

```diff
@@ -100,6 +101,8 @@ static MetricDescription metric_descriptions[] = {
   { "cgroup.cpu.usage",       "millisec", "counter", TRUE, CGROUP_SAMPLER },
   { "cgroup.cpu.shares",      "count",    "instant", TRUE, CGROUP_SAMPLER },
 
+  { "cpu.temperature", "celsius", "counter", TRUE, THERMAL_SAMPLER },
```

Comment:
It's still not a counter :)

+ please try to line it up with the ones above...